src/components/address-book/ImportDialog/index.tsx

```diff
@@ -114,52 +114,41 @@ const ImportDialog = ({ handleClose }: { handleClose: () => void }): ReactElemen
         >
           {/* https://github.com/Bunlong/react-papaparse/blob/master/src/useCSVReader.tsx */}
           {({ getRootProps, acceptedFile, ProgressBar, getRemoveFileProps, Remove }: any) => {
-            const { onClick, ...removeProps } = getRemoveFileProps()
+            const { onClick } = getRemoveFileProps()
 
             const onRemove = (e: MouseEvent<HTMLSpanElement>) => {
               setCsvData(undefined)
               setError(undefined)
               onClick(e)
             }
 
+            const fileInfo = acceptedFile
```

Comment:
Can we create a type for this and export it from `FileUpload`? It can be used here and in the `ImportAllDialog`.